unicornherder/command.py

```diff
@@ -24,6 +24,9 @@
                     help='Path to the pidfile that unicorn will write')
 parser.add_argument('-t', '--timeout', default=30, type=int, metavar='30', dest='boot_timeout',
                     help='Timeout in seconds to start workers')
+parser.add_argument('-o', '--overlap', default=120, type=int, metavar='120',
+                    dest='overlap',
+                    help='Time to wait before killing old unicorns when reloading')
```

Comment:
Would a better name for this be something like `boot_wait` or `boot_time`, as this should be the time the app takes to boot and be ready to serve requests?

overlap is a little misleading as a name because the actual overlap will be longer than the specified time. unicornherder sends SIGWINCH, and then SIGQUIT after this timeout, at which point Unicorn starts gracefully closing the old workers, but that could take some time if there are any long-running requests.